.github/workflows/laravel.yml

```diff
@@ -77,7 +77,7 @@ jobs:
         if: steps.git-diff.outputs.diff
         run: chmod -R 777 storage bootstrap/cache
       - name: Run tests
-        if: steps.git-diff.outputs.diff && matrix.php == '8.3'
+        if: steps.git-diff.outputs.diff && matrix.php != '8.4'
```

Comment:
Is this just here for the future when we're testing 8.5?

Reply:
Exactly.